x-pack/plugin/core/src/test/java/org/elasticsearch/license/AbstractLicenseServiceTestCase.java

```diff
@@ -34,7 +34,7 @@
 
 public abstract class AbstractLicenseServiceTestCase extends ESTestCase {
 
-    protected ClusterStateLicenseService clusterStateLicenseService;
+    protected LicenseService.MutableLicense licenseService;
```

Comment:
IIUC, all tests based on this abstract class make sense only for the ClusterState implementation. So maybe we could just rename this abstract class to be more specific, e.g. `AbstractClusterStateLicenseServiceTestCase`, and keep the internal unchanged.